### DISCUSSION TITLE
Can I use discordjs as a client, not a bot?

### DISCUSSION THREAD

**Discussion by 1mike12** [discordjs/discord.js#8204] [Q&A] [answered]
I don't want to do anything related to creating a bot on my own server, I just want to be able to reply to delete DMs to my personal account. 

So far everything I'm reading (despite the code naming it {Client} is pointing to the fact that this is only for admins to code up bots on their own servers?

**Comment by monbrey** [comment, marked as answer]
Automation of user accounts is known as "self-botting" and is explicitly against Discord's Terms of Service. discord.js does not support this.

**Reply by 1mike12** [reply]
ah thanks
